pvlib/bifacial.py

```diff
@@ -62,35 +63,43 @@ def pvfactors_timeseries(
         Back surface reflectivity of PV rows
     horizon_band_angle: float, default 15
         Elevation angle of the sky dome's diffuse horizon band (deg)
-    run_parallel_calculations: bool, default True
-        pvfactors is capable of using multiprocessing. Use this flag to decide
-        to run calculations in parallel (recommended) or not.
-    n_workers_for_parallel_calcs: int, default 2
-        Number of workers to use in the case of parallel calculations. The
-        '-1' value will lead to using a value equal to the number
-        of CPU's on the machine running the model.
+    run_parallel_calculations
+        Deprecated input, should be ``None``
+    n_workers_for_parallel_calcs
+        Deprecated input, should be ``None``
 
     Returns
     -------
-    front_poa_irradiance: numeric
+    poa_front: numeric
         Calculated incident irradiance on the front surface of the PV modules
         (W/m2)
-    back_poa_irradiance: numeric
+    poa_back: numeric
         Calculated incident irradiance on the back surface of the PV modules
         (W/m2)
-    df_registries: pandas DataFrame
-        DataFrame containing detailed outputs of the simulation; for
-        instance the shapely geometries, the irradiance components incident on
-        all surfaces of the PV array (for all timestamps), etc.
-        In the pvfactors documentation, this is refered to as the "surface
-        registry".
+    poa_front_absorbed: numeric
+        Calculated absorbed irradiance on the front surface of the PV modules
+        (W/m2), after AOI losses
+    poa_back_absorbed: numeric
+        Calculated absorbed irradiance on the back surface of the PV modules
+        (W/m2), after AOI losses
```

Comment:
If I understand the diff correctly, this change to the output will break user code. Ok with me but we should note it in the API Changes section. I guess there's not much point in deprecating the parallel kwargs if we're just changing the return type, so maybe best to just make users deal with the pain in the next release. I'm guessing that users of the bifacial module are relatively tolerant of api changes.

Reply:
Thanks for the feedback @wholmgren :
* I could revert the output change and return some kind of placeholder for ``df_registries`` (pvfactors v1.4.1 does not use registries anymore). Or I can just update the API changes section. Just let me know what you prefer!
* I don't mind removing the deprecation warnings as well

Reply:
In this case I think we're better off immediately conforming to the pvfactors 1.4 API, provided the API change is documented. Any objections?

Reply:
Hi @wholmgren , sorry for the delay. I removed the deprecated inputs and warnings, and updated the API changes section of the newest whatsnew file. Please let me know if you need any other fixes